### PR TITLE
Kubectl example

### PR DESCRIPTION
**Pull request by krousey**
Branched from #10595 

I removed the cmd-example in that PR because it was practically identical to the auto-generated documentation for kubectl exec. I updated the walk-throughs to point to the kubectl exec doc instead.

I also changed a few kubectl invocation examples to point to the new paths.

cc @bgrant0607 @karlkfi

### CODE REVIEW

**Comment by bgrant0607** [comment]
cc @caesarxuchao

**Comment by bgrant0607** [comment]
Thanks. Let's merge this once tests pass and then address issues like auto-TOC generation, removing references to kubectl.sh, etc. in a subsequent PR.

cc @jlowdermilk

**Comment by bgrant0607** [comment]
Please squash commits

File: docs/user-guide/walkthrough/k8s201.md [hunk 1]
```diff
@@ -20,29 +20,78 @@ certainly want the docs that go with that version.</h1>
...
+If you went through [Kubernetes 101](README.md), you learned about kubectl, pods, volumes, and multiple containers.
+For Kubernetes 201, we will pick up where 101 left off and cover some slightly more advanced topics in Kubernetes, related to application productionization, deployment and
 scaling.
 
-### Labels
-Having already learned about Pods and how to create them, you may be struck by an urge to create many, many pods.  Please do!  But eventually you will need a system to organize these pods into groups.  The system for achieving this in Kubernetes is Labels.  Labels are key-value pairs that are attached to each object in Kubernetes.  Label selectors can be passed along with a RESTful ```list``` request to the apiserver to retrieve a list of objects which match that label selector.  For example:
+In order for the kubectl usage examples to work, make sure you have an examples directory locally, either from [a release](https://github.com/GoogleCloudPlatform/kubernetes/releases) or [the source](https://github.com/GoogleCloudPlatform/kubernetes).
+
+Table of Contents
```
Comment (mikedanese) [inline docs/user-guide/walkthrough/k8s201.md]: Autogenerate?

File: docs/user-guide/walkthrough/README.md [hunk 2]
```diff
@@ -20,77 +20,176 @@ certainly want the docs that go with that version.</h1>
...
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
-# Kubernetes 101 - Walkthrough
+# Kubernetes 101 - Kubectl CLI & Pods
+
+For Kubernetes 101, we will cover kubectl, pods, volumes, and multiple containers
+
+In order for the kubectl usage examples to work, make sure you have an examples directory locally, either from [a release](https://github.com/GoogleCloudPlatform/kubernetes/releases) or [the source](https://github.com/GoogleCloudPlatform/kubernetes).
+
+Table of Contents
```
Comment (mikedanese) [inline docs/user-guide/walkthrough/README.md]: Autogenerate?

File: docs/user-guide/walkthrough/README.md [hunk 3]
```diff
@@ -20,77 +20,176 @@ certainly want the docs that go with that version.</h1>
...
+
+If you used `./cluster/kube-up.sh` to deploy your Kubernetes cluster, kubectl should already be locally configured.
+
+By default, kubectl configuration lives at `~/.kube/config`.
+
+If your cluster was deployed by other means (e.g. a [getting started guide](../../getting-started-guides/README.md)), you may want to configure the path to the Kubernetes apiserver in your shell environment:
+
+```sh
+export KUBERNETES_MASTER=http://<ip>:<port>/api
+```
```
Comment (mikedanese) [inline docs/user-guide/walkthrough/README.md]: @caesarxuchao comments that this is not usually sufficent and should point to another doc.

**Comment by krousey** [comment]
@bgrant0607 squashed

**Comment by bgrant0607** [comment]
@mikedanese I'd like to address feedback in a separate PR.

**Comment by bgrant0607** [comment]
Doc checks passed on Travis. LGTM.

**Comment by bgrant0607** [comment]
CLA bot is confused.

**Comment by krousey** [comment]
@bgrant0607 It's confused because I'm the submitter, but @karlkfi is the author.

**Comment by mikedanese** [comment]
@karlkfi needs to comment 'I'm okay with these commits being contributed' on this pr to appease googlebot

**Comment by karlkfi** [comment]
Go for it, looks like he just cherry-picked, modified and squashed from #10595

**Comment by karlkfi** [comment]
I'm okay with these commits being contributed

**Comment by karlkfi** [comment]
y'know, if the cla bot wants exact text, it should probably say what it's expecting... I've never been able to satisfy it before...

**Comment by karlkfi** [comment]
Thanks for pushing this through.

I'm not sure I completely agree with the removal of the cmd-exec example, tho. The page we're linking to now only shows that kubectl exec exists. It doesn't include an example busybox container and tell you how to actually use it...

**Comment by mikedanese** [comment]
Better to link to https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/getting-into-containers.md

**Comment by karlkfi** [comment]
That'd be an improvement, the bottom section does say how to get a bash terminal. But I still think an example exclusively for executing commands would be useful in several other cases where the docs are already giving busybox examples:
- https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/addons/dns/README.md
- https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/debugging-services.md
- https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/mesos.md

**Comment by bgrant0607** [comment]
@karlkfi Adding to getting-into-containers.md would make sense if you think a use case is missing. I'd use a container running /pause rather than sleep, however.

@mikedanese, @krousey, and/or @caesarxuchao: Please submit follow-up PRs to address the outstanding issues, like TOC generation, eliminating kubectl.sh references, linking to docs/user-guide/prereqs.md and other relevant docs in the user guide, prefixing commands with `$`, etc.

**Comment by mikedanese** [comment]
I will prepare a follow up.

**Comment by karlkfi** [comment]
@mikedanese can you link followups?

**Comment by mikedanese** [comment]
#11396

**Pull request merged**